### DISCUSSION TITLE
Best Practices for Organizing University Departments on GitHub?

### DISCUSSION THREAD

**Discussion by mohit-pesu** [community/community#137361] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi GitHub Community,

I am setting up a GitHub account for my university, and I want to effectively organize repositories by department. Ideally, I would like to have a structure that reflects different departments (like a nested organization), but I understand that GitHub doesn’t directly support this kind of hierarchy.

I've considered using Teams for each department and assigning repositories to them, but I found out that Teams are not visible to external users, which is important for us.

**Could you recommend the best practices or alternative methods for organizing university departments and their repositories on GitHub to make them easily accessible and understandable to both internal and external users?**

Thanks for your help!

**Comment by mohitpaddhariya** [comment]
# Organizing University Departments on GitHub

If you're looking to effectively organize your university's GitHub account to reflect different departments, here are some suggestions:

## Suggested Solutions

- **GitHub Teams and Visibility:** Teams within GitHub Organizations are designed for internal collaboration and are not visible to external users. Unfortunately, there isn't an option to make Teams public. They are best used for managing permissions and access control within the organization.

- **Use Repository Naming Conventions:** A practical approach is to use naming conventions for repositories that indicate the department. This method helps organize repositories logically and makes it easy for users to identify which department each repository belongs to. Examples include:
  - `biology-project-x`
  - `cs-machine-learning`
  - `math-statistics-research`

- **Using GitHub Topics:** Tagging repositories with relevant GitHub Topics can help categorize them by department. Users can then filter repositories by topics to see all related projects. This provides an easy way to organize and discover repositories across the organization.

- **Organization-Level README:** Consider creating a central `README.md` file in the root of your organization or in a separate `meta` repository. This README can provide an overview of the organizational structure, including links to each department's repositories. This approach serves as a guide for anyone visiting your GitHub organization.

- **GitHub Projects:** For more advanced organization, use GitHub Projects to manage boards for each department. Projects can link to various repositories, offering an overview and roadmap for departmental activities and initiatives. This method provides a structured way to manage and visualize work across multiple repositories.

By implementing these strategies, you can create a more organized and accessible GitHub account that reflects the different departments within your university. This will make it easier for both internal team members and external collaborators to navigate and engage with your projects.

I hope these suggestions help! If you have further questions, feel free to reach out.

**Reply by LiteBrite82** [reply, marked as answer]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
